x/launch/client/cli/tx.go

```diff
@@ -34,6 +35,8 @@ func GetTxCmd() *cobra.Command {
 	return cmd
 }
 
+const genesisURLFlag = "genesis-url"
```

Comment:
```suggestion
const genesisURLFlag = "genesis"
```

fo consistency with starport.